### ISSUE TITLE
Grafana Legacy Alerting is not working

### ISSUE THREAD

**Issue by rivalver** [VictoriaMetrics/grafana-datasource#24]
Grafana version: 9.2.4

I created 2 similar data sources but with different types (VictoriaMetrics and Prometheus).
Both are used on the same dashboards, on different panels.
Trying to create/test exactly the same alerts for each of the panels and getting the following:

Prometheus:
<img width="757" alt="image" src="https://user-images.githubusercontent.com/8620015/204789282-42ad8328-5c50-4101-80df-ccb7f1ca0c12.png">

VictoriaMetrics:
<img width="754" alt="image" src="https://user-images.githubusercontent.com/8620015/204789491-d5826921-73e8-4f63-ba4f-e9d38ab2382d.png">

The same errors in the logs:
```
logger=alerting.evalContext t=2022-11-30T11:10:13.00350573Z level=error msg="Alert Rule Result Error" ruleId=1973 name="Test alert" error="request handler error: [plugin.unavailable] plugin unavailable" changingstateto=alerting
```

**Comment by rivalver** [comment]
@dmitryk-dk thanks for the quick response. Subscribed to the PR, and will wait for the updates!

**Comment by Loori-R** [comment]
Support alerts has been included in [victoriametrics-datasource v0.1.2](https://github.com/VictoriaMetrics/grafana-datasource/releases).
